lib/rbi/parser.rb

```diff
@@ -840,6 +851,11 @@ def set_root_tree_loc
       def type_variable_definition?(node)
         node.is_a?(Prism::CallNode) && (node.message == "type_member" || node.message == "type_template")
       end
+
+      #: (Prism::Node? node) -> bool
+      def t_enum_value?(node)
+        current_scope.is_a?(TEnumBlock) && node.is_a?(Prism::ConstantWriteNode) && node.value.slice == "new"
```

Comment:
I believe the node value could be more than `new`. For example, it can be `new("Foo")`. Does this handle that case?

Reply:
Ah you're right, we need to parse it but we don't care about storing it 👍

Reply:
Done 👍